src/components/calcite-filter/calcite-filter.scss

```diff
@@ -62,12 +62,23 @@ input[type="text"]:focus {
 }
 
 .clear-button {
-  @apply text-color-2 
-    bg-none 
-    border-0 
+  @apply text-color-2
+    bg-none
+    border-0
     cursor-pointer;
   &:hover,
   &:focus {
     @apply text-color-1;
   }
 }
+
+calcite-scrim {
+  margin: -0.5rem;
```

Comment:
@julio8a is there a better way to set a negative margin of `p-2`?

Reply:
Instead of the negative margin, which might be a bit brittle, we could try top, left, right, bottom positionings of zero.

Reply:
Can't, the padding is on the host.